### PR TITLE
GitHub Sync

### PR DESCRIPTION
**Pull request by kevmoo**
This is an automatically generated pull request.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ description: >
   Requires the use of WebDriver remote server.
 homepage: https://github.com/google/webdriver.dart
 environment:
-  sdk: '>=2.0.0 <3.0.0'
+  sdk: '>=2.9.0 <2.10.0'
```
Comment (mk13) [inline pubspec.yaml:12]: According to https://dart.dev/null-safety/migration-guide#migrating-by-hand, seems like they recommend 2.12.0-0 minimum constraint?

Comment (kevmoo) [reply pubspec.yaml:12]: Yup. I'm just trying to get everything synced to github first.

Then I'll create a PR to update everything on github. Get CI working. Then send the PR to google3

Comment (mk13) [reply pubspec.yaml:12]: Gotcha, approval for now.

**Comment by mk13** [comment]
Squash and merge to sync for now. Future PR will handle Travis failures.

**Pull request merged**